pypesto/sampling/auto_correlation.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+
+
+def auto_correlation(chain: np.array):
+    '''
```

Comment:
```suggestion
    """
```
For all meant-to-be docstrings

Reply:
Right, we use double quotes throughout. Does this actually have any impact though?

Reply:
At least it's not PEP8. Not sure which tools really do care about it.